src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,10 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                .map(User::getAge)
+                .filter(user1 -> user1 >= 30 && user1 <= 45)
```

Comment:
```suggestion
                .filter(age -> age >= 30 && age <= 45)
```
이렇게 하면 코드를 이해하기 더 좋을것 같습니다.